python/cugraph-service/scripts/default-config.sh

```diff
@@ -20,19 +21,19 @@ THIS_DIR=$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)
 # file that should not be overridded by a project, then they will
 # simply not use that syntax and override, since these variables are
 # read last.
-SCRIPTS_DIR=$THIS_DIR
+export SCRIPTS_DIR=$THIS_DIR
 
 # These really should be oerridden by the project config!
-CONDA_ENV=${CONDA_ENV:-rapids}
+export CONDA_ENV=${CONDA_ENV:-rapids}
 
-GPUS_PER_NODE=${GPUS_PER_NODE:-8}
-WORKER_RMM_POOL_SIZE=${WORKER_RMM_POOL_SIZE:-12G}
-DASK_CUDA_INTERFACE=${DASK_CUDA_INTERFACE:-ib0}
-DASK_SCHEDULER_PORT=${DASK_SCHEDULER_PORT:-8792}
-DASK_DEVICE_MEMORY_LIMIT=${DASK_DEVICE_MEMORY_LIMIT:-auto}
-DASK_HOST_MEMORY_LIMIT=${DASK_HOST_MEMORY_LIMIT:-auto}
+export GPUS_PER_NODE=${GPUS_PER_NODE:-8}
+export WORKER_RMM_POOL_SIZE=${WORKER_RMM_POOL_SIZE:-12G}
+export DASK_CUDA_INTERFACE=${DASK_CUDA_INTERFACE:-ib0}
+export DASK_SCHEDULER_PORT=${DASK_SCHEDULER_PORT:-8792}
+export DASK_DEVICE_MEMORY_LIMIT=${DASK_DEVICE_MEMORY_LIMIT:-auto}
+export DASK_HOST_MEMORY_LIMIT=${DASK_HOST_MEMORY_LIMIT:-auto}
 
-BUILD_LOG_FILE=${BUILD_LOG_FILE:-${RESULTS_DIR}/build_log.txt}
-SCHEDULER_FILE=${SCHEDULER_FILE:-${WORKSPACE}/dask-scheduler.json}
-DATE=${DATE:-$(date --utc "+%Y-%m-%d_%H:%M:%S")_UTC}
-ENV_EXPORT_FILE=${ENV_EXPORT_FILE:-${WORKSPACE}/$(basename ${CONDA_ENV})-${DATE}.txt}
+export BUILD_LOG_FILE=${BUILD_LOG_FILE:-${RESULTS_DIR}/build_log.txt}
+export SCHEDULER_FILE=${SCHEDULER_FILE:-${WORKSPACE}/dask-scheduler.json}
+export DATE=${DATE:-$(date --utc "+%Y-%m-%d_%H:%M:%S")_UTC}
+export ENV_EXPORT_FILE=${ENV_EXPORT_FILE:-${WORKSPACE}/$(basename ${CONDA_ENV})-${DATE}.txt}
```

Comment:
Why the exports ?

Reply:
I didn't intend to add this to the PR, removing